### DISCUSSION TITLE
Trying to see my custom error page on my dev environment. It's coming back as an empty page

### DISCUSSION THREAD

**Discussion by cbdeveloper** [vercel/next.js#24824] [Help]
Following the official docs on: https://nextjs.org/docs/advanced-features/custom-error-page#customizing-the-500-page

I've created a `pages/500.tsx`

Something like:

```
import React from "react";
import ErrorPage from "@app/pages/error/ErrorPage";
import { getPreloadedState } from "@lib/firebase/server/getPreloadedState";
import { GetStaticProps } from "next";
import { PreloadedPageProps } from "types";

export const getStaticProps: GetStaticProps<PageErrorRoute> = async () => {
  const preloadedState = await getPreloadedState();
  return({
    props: {
      preloadedState
    }
  });
};

interface PageErrorRoute extends PreloadedPageProps {}

const PageErrorRoute: React.FC<PageErrorRoute> = (props) => {
  console.log("Rendering PageErrorRoute...");
  return(
    <ErrorPage/>
  );
};

export default React.memo(PageErrorRoute);
```

And the `<ErrorPage/>` basically has an image and some `<p>` tags saying _"We're sorry, something went wrong, etc"_.

This is my 1st project with `Next.js` and I haven't deployed it yet. So this is all being done within the `npm run dev` server.

I tried to trigger an error to force the `500` page to show.

I tried both on pages with `SSG` and `SSR` and I couldn't see my custom page. This is what come back from the error pages:

![image](https://user-images.githubusercontent.com/43407798/117169858-659ccb80-adc1-11eb-993b-1b8c3af1053b.png)

![image](https://user-images.githubusercontent.com/43407798/117170058-93821000-adc1-11eb-9f2a-29c8c2e8580a.png)

Basically it returns an empty page (it contains all the `js` bundles, preloaded data and everything), but no visible markup.

Is it supposed to show on the `dev` environment? Am I doing something wrong?
